newsfragments/1590.feature

```diff
@@ -0,0 +1 @@
+New masking parameter ``parallax_correction=``. Set to ``False`` to speed up generation of resolution masks by disabling parallax correction.
```

Comment:
Not sure, but I think a little more context here would be useful so that the general user does not think this will usefully speed up 1,800 image find spots job?